docs/changelog/75144.yaml

```diff
@@ -0,0 +1,8 @@
+pr: 75144
+summary: Security auto-configuration for packaged installations
+area: "Security"
+type: enhancement
+issues:
+ - 75704
+versions:
```

Comment:
We should remove this `versions` section here after Rory's recent refactoring.

Reply:
Thanks, I had missed that